server/controllers/overviewController.test.ts

```diff
@@ -151,16 +153,20 @@ describe('overviewController', () => {
       )
     })
 
-    it('should not call moneyService.getMoneySummary if prisoner is not in caseload', async () => {
+    it('should not call moneyService.getMoneySummary if user doesnt have money.view permission', async () => {
       const resNotInCaseload = {
```

Comment:
Was in two minds whether we could just use `res` instead of explicitly having the `resNotInCaseload`... since it's now relying on the permissions, it's not necessary is it and bloats the test... meh...

Reply:
Yeah probably right. The reason I left it was to give an idea to the reader why the permission would be false... but I guess that's not really the point of a unit test